### ISSUE TITLE
[Accessibility] VoiceOver - Unable to focus RichCombo toolbar buttons like (Font, Font Size, etc...)

### ISSUE THREAD

**Issue by sryarama** [ckeditor/ckeditor4#4855]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Launch ckeditor demo URL in IPad safari browser https://ckeditor.com/ckeditor-4/demo/ 
2. Activate VoiceOver screen reader
3. Tap on Rich Combo toolbar button.

### Actual result

Voice Over focus is immediately transferred to content editing area. Accessible users who rely on Voice Over screen reader are unable to change Font. 

## Other details
This is a regression. This was working fine in CKEditor 4.11.4 version and we recently upgraded 4.16.0 and we are observing this issue
 
* Browser: Safari
* OS: iOS
* CKEditor version: 4.16.0
* Installed CKEditor plugins: RichCombo

**Comment by jacekbogdanski** [comment]
Thanks @sryarama, indeed now the checkbox has become unusable using VoiceOver.

**Comment by jacekbogdanski** [comment]
It seems like the regression was introduced with https://github.com/ckeditor/ckeditor4/commit/1d777b1075db10434409bb7eae7dd52bb0fcb7fe

However, from what I see now the behavior of Voice Over has been rather unified with other buttons and it's still possible to open rich combo doing long tap to get proper element focus:

1. Long tap rich combo to focus it.
2. Double tap rich combo to open it.
3. Double tap rich combo option to apply changes.

https://user-images.githubusercontent.com/10279586/132189583-3af913b6-6525-4f04-a095-8a9582be39fc.mov

These steps are basically the same for any other button in the editor:
1. Long tap button to focus it.
2. Double tap the button to apply changes.

Any quick tap on the toolbar button results in focusing the editor, not the button. Due to that research, I would rather mark this ticket as a feature request to improve the behavior of Voice Over in the editor than treat it as a regression (because other buttons behave the same).

**Comment by sryarama** [comment]
Sorry for the delay response, we cannot convey that long tap to real accessibility users. If it is a regression, why can't we revert the commit which introduced this regression? Right now, accessibility users unable to use the toolbar buttons because of this issue.

**Comment by Comandeer** [comment]
It seems that the culprit is [the code located in the `toolbar` plugin](https://github.com/ckeditor/ckeditor4/blob/c0dbd14301d5639e600a6857de0478d742147342/plugins/toolbar/plugin.js#L286-L291). Ironically, it's some ancient fix for another screen reader, JAWS… As the code is 13 years old, maybe it's not needed anymore and the whole fix would be as easy as removing it.

**Comment by CKEditorBot** [comment]
Closed in https://github.com/ckeditor/ckeditor4/pull/5155
